resources/carrier/fa/98.txt

```diff
@@ -0,0 +1,30 @@
+#Copyright (C) 2016 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Prefixes from http://en.wikipedia.org/wiki/%2B98.
+
+9890|ایرانسل
+9891|همراه اول
+9892|رایتل
+98930|ایرانسل
+98931|اسپادان
+98932|تالیا
+98933|ایرانسل 
```

Comment:
Line 23 and the following Irancell lines still have trailing spaces, please remove